caluma-demo/app/templates/components/navbar.hbs

```diff
@@ -0,0 +1,9 @@
+<nav class="uk-navbar-container" uk-navbar>
+  <div class="uk-navbar-left">
+    <ul class="uk-navbar-nav">
+      <li><LinkTo @route="home">{{t "Home" }}</LinkTo></li>
+      <li><LinkTo @route="form-builder">{{t "Form-Builder" }}</LinkTo></li>
+      <li><LinkTo @route="show-forms">{{t "Show Forms" }}</LinkTo></li>
```

Comment:
Those translations won't work, you have to use keys like `nav.home` or `nav.form-builder` and then add those translations in `translations/en.yml`

Reply:
I did this already but with the wrong syntax. I have corrected it.